filters/include/pcl/filters/impl/uniform_sampling.hpp

```diff
@@ -43,19 +43,13 @@
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT> void
-pcl::UniformSampling<PointT>::applyFilter (PointCloud &output)
+pcl::UniformSampling<PointT>::applyFilter (Indices &indices)
 {
-  // Has the input dataset been set already?
-  if (!input_)
```

Comment:
Might make sense to keep this check? And clear `indices` and `removed_indices_` if input has not been set?

Reply:
https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/impl/pcl_base.hpp#L143 There is already a check in the compute initialization process, and other filters don't have this part.

Reply:
You are right, thanks!